tests/test_ilastik/widgets/test_ImageFileDialog.py

```diff
@@ -10,6 +11,9 @@
 from volumina.utility import preferences
 
 
+SKIP_ON_WIN_GH_ACTIONS = os.environ.get("GITHUB_ACTIONS", "false") == "true" and platform.system() == "Windows"
```

Comment:
Maybe split conditions and use them separately?

```python
MAC = platform.system() == "Darwin"
WIN = platform.system() == "Windows"
CI = os.environ.get("CI", "false") == "true"

@pytest.mark.skipif(MAC or WIN and CI)
```

Side note: [`CI`](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) is always implicitly defined as `true`.

Reply:
CI is also used with other CI providers (appveyor at least), and there, this test works - so I wanted to avoid using it. But yeah, I can split conditions for better readability.